### DISCUSSION TITLE
Scheduler API without job execution

### DISCUSSION THREAD

**Discussion by bcowdery** [quartznet/quartznet#1433] [Q&A] [answered]
We're building out an distributed application stack that is split into two separate deployables, a REST API front-end, and a back-end worker. This back-end worker listens to RabbitMQ and performs all of our asynchronous operations in a separate application so that we can scale the worker independently of the REST API.

We're starting to add schedule functionality, and I'd like to be able to inject the `IScheduler` into both API controllers and services used on the backend worker. But, I really only want the scheduled jobs to run on the worker with the rest of our back-end processes. 

Is it possible to register Quartz on my front-end app so that we're able to make modifications to the scheduled jobs WITHOUT jobs executing there?

Thanks!

**Comment by lahma** [comment, marked as answer]
Yes, use `ZeroSizeThreadPool` and don't start the scheduler on "maintenance node".
